docs/semgrep-assistant/getting-started.md

```diff
@@ -119,6 +119,32 @@ Semgrep Assistant messages only appear in your PR comments for rules that are se
   ![ Policies modes](/img/semgrep-assistant-comment.png)
 * You have selected PR/MR comments in **Semgrep AppSec Platform > Settings > Deployment** in the **Code** section.
 
+### Add Memories (beta)
+
+Assistant Memories allows users to tailor Assistant's remediation guidance to their organization's standards and defaults on a per-project, per-rule basis. Whenever Assistant gives a suggested fix,  users can provide feedback by adding custom instructions.
+
+Memories are enabled by default for all Assistant users.
+
+To add a Memory:
+
+1. Identify the specific instance of **Assistant's suggested fix** that you want to modify. Click **Customize fix** to open an input box, and enter your preferred remediation approaches and secure defaults for the project.
```

Comment:
How prescriptive does the submitted, customized fix need to be? I wonder if the user can enter more general instructions like, “Fix it by using X” rather than explicitly writing the code that should replaced in line. I think it’s the former based on what I’ve seen and if that’s true, I think it would be valuable to explain that here.

Reply:
Yes, people can enter fairly general instructions, so I'll add an example of that